src/main/java/com/redhat/devtools/intellij/lsp4ij/LanguageServersRegistry.java

```diff
@@ -45,10 +44,13 @@ public abstract static class LanguageServerDefinition {
 
         private static final int DEFAULT_LAST_DOCUMENTED_DISCONNECTED_TIMEOUT = 5;
 
-        public final @Nonnull String id;
-        public final @Nonnull String label;
+        public final @Nonnull
+        String id;
+        public final @Nonnull
+        String label;
         public final boolean isSingleton;
-        public final @Nonnull Map<Language, String> languageIdMappings;
+        public final @Nonnull
+        Map<Language, String> languageIdMappings;
```

Comment:
can you keep the declarations on a single line?

Reply:
fixed, but it is format code of IJ which does that